src/resource/image.rs

```diff
@@ -210,26 +213,71 @@ impl<A: Allocator> Image<A> {
     }
 
     /// Construct a trivial [`ImageView`] from this [`Image`]. This is an image view that views the
-    /// entire image subresource.
+    /// whole image subresource.
+    /// * `aspect` - The image aspect flags that will be used to create the image view
     /// <br>
     /// <br>
     /// # Lifetime
     /// The returned [`ImageView`] is valid as long as `self` is valid.
-    pub fn view(&self, aspect: vk::ImageAspectFlags) -> Result<ImageView> {
+    pub fn whole_view(
```

Comment:
If `whole_view` should always view the entire image, then passing the type is unnecessary because we can find what it should be based on the dimensions (`depth > 1` means 3D, otherwise 2D).

Reply:
Right I wanted to add that but I didn't want to assume that users will be doing that. Will fix